queries/haskell/injections.scm

```diff
@@ -2,11 +2,11 @@
 ;; General language injection
 
 (quasiquote
- ((quoter) @language)
- ((quasiquote_body) @content)
+ ((quoter) @injection.language)
+ ((quasiquote_body) @injection.content)
 )
 
-(comment) @comment
+((comment) @injection.content (#set! injection.language "comment"))
 
 ;; -----------------------------------------------------------------------------
 ;; shakespeare library
```

Comment:
You should also update the comment for consistency.

Reply:
Agreed
Was just :s -ing and did not look at the style that much.
I will update in following PRs

Reply:
I meant this one. It's not in the diff.
https://github.com/nvim-treesitter/nvim-treesitter/blob/8e0e12f3dc32539928da01051d6ebb2423b5cfd2/queries/haskell/injections.scm#L15-L20